### PR TITLE
Add cdn-edges.net

### PR DESCRIPTION
**Pull request by tomkln**
* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

Description of Organization
====

Organization Website: https://www.gentlent.com

I am a director and engineer at Gentlent, which is a software development and consulting company.

Reason for PSL Inclusion
====

It's about cookie security, as we'll provide subdomains to our customers as part of our in-dev CDN offering.

DNS Verification via dig
=======

```
dig +short TXT _psl.cdn-edges.net
"https://github.com/publicsuffix/list/pull/1110"
```

make test
=========

We ran the test.

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11871,6 +11871,7 @@ usercontent.jp
 
 // Gentlent, Inc. : https://www.gentlent.com
 // Submitted by Tom Klein <tom@gentlent.com>
+cdn-edges.net
```
Comment (dnsguru) [inline public_suffix_list.dat:11874]: @tomkln sorry to pedant this, please revise your change to organize the entries in your section ordered alphabetically by TLD then SLD ascending (.net after .com and .ms)

Comment (tomkln) [reply public_suffix_list.dat]: Thanks for letting me know. I forgot that. **It's fixed** :)

**Review by dnsguru — APPROVED** [review]
- DNS Validates
- No Conflict with Base
- Tests Pass
- Rationale ok

**APPROVED**

**Comment by dnsguru** [comment]
@tomkln thank you for revising the sorting;  Approved and merged

**Comment by wdhdev** [comment]
@tomkln Are the domains in the `Gentlent, Inc.` block still required? I cannot seem to find any usage on them. 

If we don't receive a response with-in an extended period of time (around 30 days) your entries may be removed.

cc @groundcat

**Pull request merged**